### PR TITLE
feat: update saved chart name and description in explore page

### PR DESCRIPTION
**Pull request by ZeRego**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1819 

### Scope of the changes (select all that apply):
- [x] Frontend  
- [ ] Backend  
- [x] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: packages/frontend/src/components/Explorer/ExploreHeader/ExploreHeader.tsx [hunk 1]
```diff
@@ -30,14 +31,45 @@ const ExploreHeader: FC = () => {
...
+                                            marginLeft: 5,
+                                        }}
+                                        icon="info-sign"
+                                        minimal
+                                    />
+                                </Tooltip2>
+                            )}
+                            <Button
+                                style={{
+                                    marginLeft: 5,
```
Comment (rephus) [inline packages/frontend/src/components/Explorer/ExploreHeader/ExploreHeader.tsx:60]: inline styles ?

File: packages/frontend/src/components/SavedQueries/RenameSavedChartModal.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                <div className={Classes.DIALOG_FOOTER}>
+                    <div className={Classes.DIALOG_FOOTER_ACTIONS}>
+                        <Button onClick={onClose}>Cancel</Button>
+                        <Button
+                            data-cy="submit-base-modal"
+                            intent={Intent.SUCCESS}
+                            text="Save"
+                            type="submit"
+                            onClick={(e) => {
+                                mutate({ name, description });
```
Comment (rephus) [inline packages/frontend/src/components/SavedQueries/RenameSavedChartModal.tsx:70]: don't we need to check if name != '' ?

Comment (ZeRego) [reply packages/frontend/src/components/SavedQueries/RenameSavedChartModal.tsx:70]: I copy pasted the entire form from the creation modal but now I'm changing it to have the latest form component, which include validation.

**Comment by ZeRego** [comment]
> If you add a space to description, you can see the (i) button, but no tooltip. I guess that's ok.

imo, not worth handling those edge cases

> Am I the only one who gets a weird blue border when clicking on any of those 2 buttons ?

It also happens to me. Blueprint highlights the selected button for accessibility purposes and the header div has overflow:none so the top&bottom parts are cutoff.

**Review by rephus — APPROVED** [review]

**Pull request merged**